### ISSUE TITLE
Explanation on how to use options

### ISSUE THREAD

**Issue by janpilu** [TGM-HIT/diploma-thesis#10]
A quick explanation on how to apply the described options in the README.md would be helpful.
As our team struggled a long time with finding out how to change the studys language to english, adding something like the following lines into the README.md could make future students lifes easier.
````markdown
The following options can be enabled by adding them as parameters in the 1st line of the main.tex as shown below:
​```latex
\documentclass[<option1>,<option2>,...]{thesis}
​```
````

**Comment by re1** [comment]
Good idea and thank you for the feedback. I will include some simple instructions based on your example and also add them to the [latex-protocol](github.com/tgm-hit/latex-protocol) repository as it uses the same concept.

**Comment by re1** [comment]
Fixed trough 5700ce5d61ab35a5c909ab40d306f0b2fee100ff.
